lib/tasks/build-sass.js

```diff
@@ -31,6 +34,24 @@ function getSassData(sassFile, config = {
 	);
 }
 
+/**
+ * Build Sass and return CSS.
+ *
+ * @param {Object} config - Configuration to build sass.
+ * @param {String} config.cwd - The working directory to resolve assets paths.
+ * @param {String} config.sass ['main.scss'] - The Sass file to build.
+ * @param {String} config.buildFolder ['/build'] - The destination for build assets. Either a directory or "disabled", to never write build css to file.
+ * @param {String} config.buildCss ['main.css'] - The destination filename.
+ * @param {Boolean} config.sourcemaps [true] - Whether to include inline sourcemaps (defaults to `false` when `production` is `true`).
+ * @param {Boolean} config.production [false] - Whether this is a production build. Non-production builds do not minify CSS and include sourcemaps by default.
+ * @param {String|Undefined} config.brand [undefined] - The brand the Sass build is for .e.g. "master", "internal", or "whitelabel".
+ * @param {String|Undefined} config.sassPrefix [undefined] - Sass to prefix the Sass from file with.
+ * @param {String} config.outputStyle ['expanded'] - The Sass output style. One of `compressed` (removes as many extra characters as possible) or `expanded` (writes each selector and declaration on its own line).
+ * @param {Boolean} config.verbose [false] - When true, Sass warnings and debug messages are output.
+ * @param {String[]} config.sassIncludePaths - Extra Sass paths to includes.
+ * @param {*} config.ignoreBower - True to include npm Sass paths, otherwise bower paths are used.
+ * @return {String} - The built css.
+ */
```

Comment:
This JSDoc is too good @notlee

Reply:
Thanks! This method could use more work, but I wanted to documented what was there to make that easier in the future